src/overlays/actors/ovl_Door_Warp1/z_door_warp1.c

```diff
@@ -579,8 +579,8 @@ void func_80999FE4(DoorWarp1* this, GlobalContext* globalCtx) {
     if (Message_GetState(&globalCtx->msgCtx) == TEXT_STATE_NONE) {
         Audio_PlaySoundGeneral(NA_SE_EV_LINK_WARP, &this->actor.projectedPos, 4, &gSfxDefaultFreqAndVolScale,
                                &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
-        OnePointCutscene_Init(globalCtx, 0x25E9, 999, &this->actor, MAIN_CAM);
-        Gameplay_CopyCamera(globalCtx, -1, sRutoWarpSubCamId);
+        OnePointCutscene_Init(globalCtx, 0x25E9, 999, &this->actor, CAM_ID_MAIN);
+        Gameplay_CopyCamera(globalCtx, CAM_ID_NONE, sRutoWarpSubCamId);
         Gameplay_ChangeCameraStatus(globalCtx, sRutoWarpSubCamId, CAM_STAT_WAIT);
         this->rutoWarpState = WARP_BLUE_RUTO_STATE_WARPING;
```

Comment:
perhaps `// passing `CAM_ID_NONE` here forces it to default to the active camera`, and in the other actor that does this

Reply:
Specifically for `Gameplay_CopyCamera` or for all instances of `CAM_ID_NONE`, or a middle ground where the intent was for it to be "Active"

Reply:
just the two that used to be CAM_ID_ACTIVE or whatever, before we reverted it
think those were the only two cases